src/components/App.tsx

```diff
@@ -2576,6 +2609,26 @@ class App extends React.Component<ExcalidrawProps, AppState> {
     return false;
   };
 
+  private isHittingASelectedElement(
+    hitElement: ExcalidrawElement | null,
+  ): boolean {
+    return hitElement != null && this.state.selectedElementIds[hitElement.id];
+  }
+
+  private isShiftKeyPressed(
+    event: React.PointerEvent<HTMLCanvasElement>,
+  ): boolean {
+    return event.shiftKey;
+  }
```

Comment:
Not a fan of these micro functions that introduce more cognitive load & worse performance than simply doing `event.shiftKey` in place.

Reply:
I agree with you. I ended up doing that because, previously, there was a comment explaining what `event.shiftKey` did. I tend to prefer explaining with function names than with comments whenever possible. Either way, I've inlined that :)